troposphere/static/js/components/modals/instance/launch/components/BootScriptOption.react.js

```diff
@@ -0,0 +1,67 @@
+
+import React from 'react/addons';
+import AddScripts from './AddScripts.react';
+import CreateScript from './CreateScript.react';
+
+
+export default React.createClass({
+
```

Comment:
Is there a reason a significant major of newly defined components have no `propTypes` defined? 

They are optional (in the sense React works without them). I do think we want to define them to get the added validation from warning that React will generate for us.